geoportal/c2cgeoportal_geoportal/lib/caching.py

```diff
@@ -193,9 +193,10 @@ def delete(self, key: str) -> None:
 
 
 class Cache(Enum):
-    NO = 0
-    PUBLIC = 1
+    PUBLIC = 0
+    PUBLIC_NO = 1
```

Comment:
Not sure to understand, what does PUBLIC_NO stand for ?
It seems is is for something like: `Cache-Control: public, max-age=10`
Maybe this desserve some comments.

Reply:
Comment added :-)